doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -1065,6 +1081,51 @@ Generation of induction principles with ``Scheme``
   The type of `even_odd` shares the same premises but the conclusion is
   `forall n : nat, even n -> P0 n`.
 
+.. example:: Example of different `Scheme` commands
+
+   Let us demonstrate the difference between the Scheme commands.
+
+   .. coqtop:: in
+
+      Unset Elimination Schemes.
+
+      Inductive Nat :=
+      | z : Nat
+      | s : Nat -> Nat.
+
+      Scheme Induction for Nat Sort Set.
+
+      About Nat_rec.
+
+      (*
+      Nat_rec :
+      forall P : Nat -> Set,
+      P z ->
+      (forall n : Nat, P n -> P (s n)) -> forall n : Nat, P n
+      *)
```

Comment:
I think you'll see these comments appear in the output and thus should go away.  Might be nice to add comments like `(* recursive, non-dependent *)` on each of the 4 to be really clear.